extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -238,30 +241,18 @@ export async function createAzureFunction(node?: ITreeNodeInfo): Promise<void> {
 				...(selectedBindingType === BindingType.output && { table: objectName })
 			},
 			folderPath: projectFolder,
-			suppressCreateProjectPrompt: true
+			suppressCreateProjectPrompt: true,
+			...(isCreateNewProject && { executeStep: connectionStringExecuteStep })
```

Comment:
So one thing you may want to do is update this to actually take in an array of items (or even better either single or array). That way if we have more steps we want to add later then we don't have to get another version of the extension out.

Reply:
I can look into adding this for future cases as well.